README.md

```diff
@@ -12,6 +12,12 @@ Requirements
 * PHP 7.3 or 7.4
 * [Composer](http://getcomposer.org/)
 * The `fc-list` command
+* The following fonts:
+  * `fonts-freefont-ttf`
+  * `fonts-linuxlibertine`
+  * `fonts-dejavu-core`
+  * `fonts-gubbi`
+  * `fonts-opendyslexic`
```

Comment:
Maybe state that these fonts are optional and ws-export can work without them?

Reply:
Yet, they are needed if we want the tests (`FontProviderTest.php`) to pass. I'll add a note :wink: